### PR TITLE
Newer version of Jyutping tables

### PR DESCRIPTION
**Pull request by mahiuchun**
Source: https://github.com/kcwu/xcin25/tree/master/cin/big5

### CODE REVIEW

**Comment by mahiuchun** [comment]
Ping? This might be a trivial/meaningless change. But I also got major change for jyutping.

**Comment by lukhnos** [comment]
This looks good except one issue: Have you tried to use the updated jyutping.cin in OpenVanilla? The cin parser in OV uses everything in the definition component, including the comments, and so I think you'll have to remove the inline comments for those to be useful in OV.

**Comment by mahiuchun** [comment]
I guess you are right. Let me re-write history a little bit.

File: DataTables/jyutping.cin [hunk 1]
```diff
@@ -12677,4 +12678,718 @@ zyut6   拙
...
+aa2    瘂
+but6   癶
+gou1   㟸
+zan1   眞
+zoek6  着
+zoek3  着
+zaau1  着
+zaau2  着
+si1    㺨
+#
```
Comment (lukhnos) [inline DataTables/jyutping.cin]: I think you also have to move those comments out of the `%chardef` section or remove them entirely–for efficiency reasons OV parses the comments inside `%chardef` as actual definitions (so the comments become definitions of the key `#`).

**Comment by lukhnos** [comment]
@mahiuchun One more issue on the comments inside the `%chardef` section and this will be good. Thanks!

**Comment by mahiuchun** [comment]
Done

**Comment by lukhnos** [comment]
@mahiuchun Thank you!

**Comment by lukhnos** [comment]
The two .cin tables are now built-in as of 1.3.0: https://github.com/openvanilla/openvanilla/releases/tag/1.3.0

Thanks for the contribution!

**Pull request merged**